.github/ISSUE_TEMPLATE/propose-new-topic.md

```diff
@@ -1,20 +1,21 @@
 ---
 name: Propose a new topic
 about: Suggest the addition of a new page or section
-title: ''
-labels: 'enhancement'
-assignees: ''
-
+title: ""
+labels: "enhancement"
+assignees: ""
 ---
 
 ## Summary
+
 A short description of what you'd like to see covered
 
 ## Background info
 
 What do you already know about the topic? What would you like to learn? What sort of things will this information help you to do?
 
-(If you're proposing this topic because you think it would benefit *someone else*, then summarize *their* background knowledge and needs)
+(If you're proposing this topic because you think it would benefit _someone else_, then summarize _their_ background knowledge and needs)
```

Comment:
Is there a way to NOT do this (underscores for emphasis)? Changes a lot of files.

Reply:
Not that I can see: https://prettier.io/docs/en/options.html. I personally like the fact that Prettier is so opinionated so we avoid this type of bike shedding.

Reply:
Nice long-term, just a lot of changes to read through :)

Reply:
Yeah, why I wanted to isolate this specific PR, pay the one time price. Not sure why they weren't caught before by husky (maybe that came later).